packages/engine-server/lib/adapters/minio.js

```diff
@@ -49,6 +50,29 @@ async function saveSignature({
   )
 }
 
+function getObject(bucket, name) {
+  return new Promise(async (resolve, reject) => {
+    const stream = await minioClient.getObject(bucket, name)
+    let chunks = ''
+    stream.on('data', (data) => {
+      chunks += data.toString()
+    })
+    stream.on('end', () => {
+      resolve(JSON.parse(chunks))
+    })
+    stream.on('error', reject)
+  })
+}
+async function getSignatures() {
```

Comment:
```suggestion
function getSignatures() {
```
Not needed since you don't use `await` inside the function